resource-definitions/sconfig-example3.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: "secretless.io/v1"
```

Comment:
maybe this file should be called `sconfig-example2-revised` to clarify its intended use?